Gemfile

```diff
@@ -75,6 +75,7 @@ group :test do
   gem 'gherkin'
   gem 'guard-rails'
   gem 'guard-rspec'
+  gem 'factory_girl'
```

Comment:
We already have `factory_girl_rails` in Gemfile, so I think you do not need to add a new gem here.